gulpfile.js

```diff
@@ -37,5 +38,10 @@ gulp.task('tslint', () =>
         .pipe(tslint.report())
 );
 
-gulp.task('build', [ 'typescript', 'copydecs' ]);
+gulp.task('typedoc', () => {
+    exec('`npm bin`/typedoc --name "Resin ProcBots" --module commonjs --target ES6 --includeDeclarations ' +
+        '--excludeExternals --out docs/Typedoc lib/');
```

Comment:
Even if we include declarations, we should include `typings` as externals, since we _definitely_ don't want docs for those, and by changing the root they mess up all the file paths shown (making them lib/X and typings/Y, rather than just X). `--externalPattern '**/typings/*.d.ts'` seems to work (although it's strange and annoying to me that `**` is required there)